src/screens/LoginScreen/Login.tsx

```diff
@@ -5,21 +5,21 @@ import {
     TouchableOpacity,
     Image,
 } from 'react-native';
-import React, {useEffect, useRef} from 'react';
+import React, {useEffect} from 'react';
 import {useDispatch, useSelector} from 'react-redux';
 import {loginUser} from "../../redux/actions/loginAction";
 import {loadUser} from "../../redux/actions/loadUser";
 import {useNavigation} from "@react-navigation/native";
 import {Formik} from "formik";
 import {styles} from "./style";
 import {loginValidationSchema} from "./validationScheme";
-import Animated, {Easing, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
-import Toast from "../../components/Toast";
+import Animated from 'react-native-reanimated';
 import {resetError} from "../../redux/actions/resetError";
 import {displayErrorMessage} from "../../utils/displayMessage";
 import {RootState} from "../../redux/reducers/rootReducer";
 import {AnimatedText} from "./AnimatedText";
 import {AnimatedBackground} from "./AnimatedBackground";
+import {colors} from "../../../assets/colors/colors";
 
```

Comment:
`displayErrorMessage` is used here no matter was it actually an error or a success case. Should be renamed